### DISCUSSION TITLE
jjb

### DISCUSSION THREAD

**Discussion by organboy1024** [GrandOrgue/grandorgue#1683] [Help]
grandorgue will not repond to midi, keys all sound! are stuck! did not have this problem before?

**Comment by rousseldenis** [comment]
> grandorgue will not repond to midi, keys all sound! are stuck! did not have this problem before?

@organboy1024 Could you elaborate a little bit your problem?

Which version of GO do you use? On what system?

**Comment by organboy1024** [comment]
Hi Dennis I use version v 0.3.1.2330, windows 10, hope you can help thanks!  I have Errors: Failed to read directory 'C:\Program Files (x86)\GrandOrgue\share\GrandOrgue\packages'
 'C:\Users\JEFF\AppData\Roaming\GrandOrgueCache\6CB15696419D4E412B4CCCFDB85041F427AF630F.idx' has bad magic - bypassing index9/29/2023 10:16:21 AM: 10:16:21 AM: Error: File 'sounds\metronome\beat.wv' does not exists9/29/2023 10:16:27 AM: 10:16:27 AM: Error: Error while loading samples for rank Metronome pipe sounds\metronome\beat.wv: Failed to open file 'sounds\metronome\beat.wv'
9/29/2023 10:16:57 AM: 10:16:26 AM: Error: caught exception: Failed to open file 'sounds\metronome\beat.wv'
9/29/2023 10:18:19 AM: 10:18:19 AM: Error: File 'sounds\metronome\beat.wv' does not exists9/29/2023 10:18:19 AM: 10:18:19 AM: Error: File 'sounds\metronome\first_beat.wv' does not exists9/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Unused ODF entry 'Organ/PitchCorrection'9/29/2023 10:18:22 AM: 10:18:22 AM: Error: Memory mapping of the cache file failed with error code 89/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Cache file had diffent hash bypassing cache.
9/29/2023 10:18:22 AM: 10:18:22 AM: Error: Memory mapping of the cache file failed with error code 89/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Cache file had diffent hash bypassing cache.-------------------- From: "Denis Roussel (ACSONE)" ***@***.***> Reply-To: "GrandOrgue/grandorgue" ***@***.***> To: "GrandOrgue/grandorgue" ***@***.***> Cc: "organboy1024" ***@***.***>, "Mention" ***@***.***> Sent: Thu, Sep 28 2023 03:29 AM Subject: Re: [GrandOrgue/grandorgue] jjb (Issue #1670)
 

grandorgue will not repond to midi, keys all sound! are stuck! did not have this problem before?

@organboy1024 Could you elaborate a little bit your problem?
Which version of GO do you use? On what system?

**Comment by oleg68** [comment]
@organboy1024 is the problem reproducible or it appeared only once?

**Comment by organboy1024** [comment]
Hi, Never had the problem before! will not detect midi keyboard, all keys stuck when played with mouse! hope u can help love the organs! use version v 0.3.1.2330, windows 10, hope you can help thanks!  errors: Failed to read directory 'C:\Program Files (x86)\GrandOrgue\share\GrandOrgue\packages'
 'C:\Users\JEFF\AppData\Roaming\GrandOrgueCache\6CB15696419D4E412B4CCCFDB85041F427AF630F.idx' has bad magic - bypassing index9/29/2023 10:16:21 AM: 10:16:21 AM: Error: File 'sounds\metronome\beat.wv' does not exists9/29/2023 10:16:27 AM: 10:16:27 AM: Error: Error while loading samples for rank Metronome pipe sounds\metronome\beat.wv: Failed to open file 'sounds\metronome\beat.wv'
9/29/2023 10:16:57 AM: 10:16:26 AM: Error: caught exception: Failed to open file 'sounds\metronome\beat.wv'
9/29/2023 10:18:19 AM: 10:18:19 AM: Error: File 'sounds\metronome\beat.wv' does not exists9/29/2023 10:18:19 AM: 10:18:19 AM: Error: File 'sounds\metronome\first_beat.wv' does not exists9/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Unused ODF entry 'Organ/PitchCorrection'9/29/2023 10:18:22 AM: 10:18:22 AM: Error: Memory mapping of the cache file failed with error code 89/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Cache file had diffent hash bypassing cache.
9/29/2023 10:18:22 AM: 10:18:22 AM: Error: Memory mapping of the cache file failed with error code 89/29/2023 10:18:22 AM: 10:18:22 AM: Warning: Cache file had diffent hash bypassing cache.
-------------------- From: "Oleg Samarin" ***@***.***> Reply-To: "GrandOrgue/grandorgue" ***@***.***> To: "GrandOrgue/grandorgue" ***@***.***> Cc: "organboy1024" ***@***.***>, "Mention" ***@***.***> Sent: Fri, Sep 29 2023 12:12 PM Subject: Re: [GrandOrgue/grandorgue] jjb (Issue #1670)
 
@organboy1024 is the problem reproducible or it appeared only once?

**Comment by oleg68** [comment]
One of the error message is caused by the ODF is not compatible with the old GrandOrgue version. Another one is caused by you have deleted the `C:\Program Files (x86)\GrandOrgue` directory where the organ was loaded from.

I suggest you to reinstall a new GrandOrgue version from scratch:

- Remove the `C:\Users\JEFF\AppData\Roaming\GrandOrgueConfig` file that contains the GrandOrgue settings
- Download and install the new GrandOrgue version from https://github.com/GrandOrgue/grandorgue/releases/download/3.12.3-1/grandorgue-3.12.3-1.windows.x86_64.exe
- Try to reproduce the problem
